### PR TITLE
Update FreeBSD installation process

### PR DESCRIPTION
**Pull request by xmacan**
There is a lot obsolete information about FreeBSD. I made little update.

### CODE REVIEW

File: General-Installing-Instructions.md [hunk 1]
```diff
@@ -58,16 +58,43 @@ The following modules are optional, but preferred to be installed.
...
-- `net/net-snmp`
-- `www/php-cgi`
-- `lang/php` (With MySQL and SNMP Support)
-- `databases/mysql-server`
+- FreeBSD ports - compilation could last long time, but without invariant dependencies
+
+For both ways, cacti has a lot of dependent packages, you don't need to install anything else. Everything is prepared.
+
+- For packages you need only:
+```sh
```
Comment (netniV) [inline General-Installing-Instructions.md]: This code block should all be indented by three characters to aline with the bullet point.

Comment (netniV) [inline General-Installing-Instructions.md]: This code block should all be indented by three characters to aline with the bullet point.

File: General-Installing-Instructions.md [hunk 2]
```diff
@@ -58,16 +58,43 @@ The following modules are optional, but preferred to be installed.
...
+pkg install cacti
+pkg install spine
+```
+
+- For ports:
+[Howto use ports](https://www.freebsd.org/doc/handbook/ports-using.html) 
+```sh
```
Comment (netniV) [inline General-Installing-Instructions.md]: - There should be a blank line between text and code block.  
- Both the text and code block should be indented to align with the bullet point.

File: General-Installing-Instructions.md (hunk 1 above)
Comment (xmacan) [reply General-Installing-Instructions.md]: I changed it more. It is better now

File: General-Installing-Instructions.md (hunk 2 above)
Comment (xmacan) [reply General-Installing-Instructions.md]: done

**Comment by netniV** [comment]
There are still a couple of issues, like some styling and that this new file isn't linked from the index page, but I'll fix that up after.

**Pull request merged**